cmake/External/nccl.cmake

```diff
@@ -38,6 +42,7 @@ if(NOT __NCCL_INCLUDED)
       BUILD_IN_SOURCE 1
       CONFIGURE_COMMAND ""
       BUILD_COMMAND
+        env
```

Comment:
Why this is needed?

Reply:
It was there before, but removed by #83696. I thought we need to pass this to pass env vars to make if necessary?

Reply:
No `env` vars are being passed though, it was left over from the `CCACHE_DISABLE=1` that was there before.

Reply:
Oh I see, let me remove that then.